src/App.js

```diff
@@ -39,7 +29,7 @@ const App = () => {
 
                 const globalFilterConfig = mapGlobalFilters(TAGs, SIDs, data?.Workloads);
 
-                if (!isEqual(config, globalFilterConfig)) {
+                if (JSON.stringify(config) === JSON.stringify(globalFilterConfig)) {
```

Comment:
I am again using `JSON.stringify` for deep comparison. For example, `JSON.stringify({a: 42, b: 0}) === JSON.stringify({b: 0, a: 42})` results to false. https://stackoverflow.com/questions/201183/how-to-determine-equality-for-two-javascript-objects#comment82819893_46645497

Lodash is making this job perfectly.

Reply:
The expected false in your example is what I want. The idea behind deep comparisons is also somehow avoiding this extra trigger. I am interested in the value pairs not their order in the object.

Reply:
aha, in that case this should be fine. (though you are technically adding another requirement for this variable to be always the same order; this can complicate refactor/debugging in the future)